metrics/internal/collectors/rbd-mirror.go

```diff
@@ -87,10 +59,6 @@ func (c *RBDMirrorCollector) Describe(ch chan<- *prometheus.Desc) {
 	ds := []*prometheus.Desc{
 		c.MirrorDaemonHealth,
 		c.ImageStatusState,
```

Comment:
Are these metrics not exported by Ceph?

Reply:
No, these metrics are not exported as metrics. Just in the status, as the status is calculated in a different layer than the daemon.